src/pages/wishlistpage/Wishlist.jsx

```diff
@@ -1,15 +1,29 @@
 import "./Wishlist.css";
 import { ProductCard } from "../../components";
+import { useEffect } from "react";
+import { getWishlist } from "../../api/apicall";
+import { useProductContext } from "../../context/ProductContext";
 const Wishlist = () => {
+	const { productState, productDispatch } = useProductContext();
+
+	useEffect(() => {
+		const getWishlistData = async () => {
+			const response = await getWishlist();
+			response.success
+				? productDispatch({ type: "SET_WISHLIST", payload: response.wishlist })
+				: productDispatch({ type: "SET_WISHLIST", payload: [] });
+		};
+		getWishlistData();
+	}, [productDispatch]);
 	return (
 		<div className="wishlist-page">
-			<p className="typo-title flex-hz-center p-2">My Cart</p>
-			<div class="grid grid-4-responsive">
-				<ProductCard card_type={"wishlist_card"} />
-				<ProductCard card_type={"wishlist_card"} />
-				<ProductCard card_type={"wishlist_card"} />
-				<ProductCard card_type={"wishlist_card"} />
-				<ProductCard card_type={"wishlist_card"} />
+			<p className="typo-title flex-hz-center p-2">
+				My Wishlist {"(" + productState.wishlist.length + " items)"}
```

Comment:
You could consider using template literals here

Reply:
used template literals.